topics/qt/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: qt-framework, cute, qt-application
+created_by: Haavard Nord, Eirik Chambe-Eng
+display_name: Qt
+github_url: https://github.com/qt
+logo: Qt_logo.png
+released: 20 May 1995
+short_description: Qt is a cross-platform application framework that is used for developing application software.
+topic: qt
+related: cpp
+url: https://www.qt.io/
+wikipedia_url: https://en.wikipedia.org/wiki/Qt_(software)
+---
+Qt is a cross-platform application network that is used for developing application that can run on various hardware and software platforms. Qt is available with both proprietary and open source licenses.
```

Comment:
> application network

Is this right?

> can run on various hardware and software platforms

You could list a few:

> can run on various hardware and software platforms, including Linux, macOS, iOS, Windows, and Blackberry.

Source: http://wiki.qt.io/About_Qt